package.json

```diff
@@ -6,7 +6,7 @@
     "ios": "expo start --ios",
     "web": "expo start --web",
     "eject": "expo eject",
-    "test": "jest --forceExit",
+    "test": "jest --update-snapshot --forceExit",
```

Comment:
Very cool! We should also look into adding pre-commit files locally just in case.